src/Murder.Editor/Systems/EditorSystem.cs

```diff
@@ -138,8 +138,8 @@ public void DrawGui(RenderContext render, Context context)
 
                     if (_showRenderInspector && ImGui.Begin("Render Inspector", ref _showRenderInspector, ImGuiWindowFlags.None))
                     {
-                        (bool mod, _inspectingRenderTarget) = ImGuiHelpers.DrawEnumField("Render Target", typeof(RenderContext.RenderTargets), _inspectingRenderTarget);
-                        var image = render.GetRenderTargetFromEnum((RenderContext.RenderTargets)_inspectingRenderTarget);
+                        (bool mod, object inspectingRenderTargetObject) = ImGuiHelpers.DrawEnumField("Render Target", typeof(RenderContext.RenderTargets), _inspectingRenderTarget);
```

Comment:
should also not be needed?

Reply:
This is to fix the previous DrawEnumField error, and there is no need for it anymore